hbase-rsgroup/src/test/java/org/apache/hadoop/hbase/rsgroup/TestRSGroupUtil.java

```diff
@@ -57,39 +68,40 @@ public static void setUp() throws Exception {
       RSGroupAdminEndpoint.class.getName());
     UTIL.startMiniCluster(5);
     master = UTIL.getMiniHBaseCluster().getMaster();
-    admin = UTIL.getAdmin();
-    rsGroupAdminClient = new RSGroupAdminClient(UTIL.getConnection());
 
-    HRegionServer rs = UTIL.getHBaseCluster().getRegionServer(0);
-    rsGroupAdminClient.addRSGroup(GROUP_NAME);
-    rsGroupAdminClient.moveServers(Collections.singleton(rs.getServerName().getAddress()), GROUP_NAME);
-    admin.createNamespace(NamespaceDescriptor.create(NAMESPACE_NAME)
-      .addConfiguration(RSGroupInfo.NAMESPACE_DESC_PROP_GROUP, GROUP_NAME)
-      .build());
+    UTIL.waitFor(60000, (Predicate<Exception>) () ->
+        master.isInitialized() && ((RSGroupBasedLoadBalancer) master.getLoadBalancer()).isOnline());
 
-    rsGroupInfoManager = RSGroupInfoManagerImpl.getInstance(master);
-    rsGroupInfoManager.start();
+    rsGroupAdminClient = new RSGroupAdminClient(UTIL.getConnection());
+
+    List<RSGroupAdminEndpoint> cps =
+        master.getMasterCoprocessorHost().findCoprocessors(RSGroupAdminEndpoint.class);
+    assertTrue(cps.size() > 0);
+    rsGroupInfoManager = cps.get(0).getGroupInfoManager();
   }
 
   @AfterClass
-  public static void tearDown() throws IOException {
-    admin.deleteNamespace(NAMESPACE_NAME);
```

Comment:
Why this removed?

Reply:
namespace creation is redundant in this UT, so I removed the code related namespace